.github/workflows/publish-on-fleek.yaml

```diff
@@ -4,20 +4,18 @@ on:
   workflow_dispatch:
     inputs:
       source_branch:
-        description: "Publish content from branch (master|staging)"
+        description: 'Publish content from branch (master|staging-temp)'
         required: true
-        default: staging
+        default: staging-temp
         type: choice
         options:
-        - master
-        - staging
+          - master
```

Comment:
make sure the indention here is correct

Reply:
prettier is adding the space automatically when saving